pwa-devdocs/src/_data/product-recs.yml

```
@@ -0,0 +1,4 @@
+title: Product Recommendations
+entries:
+    - label: Integrating Product Recommendations
```

```suggestion
    - label: Integration guide
```
we're already in the Product Recommendations section, so there's probably no need to include it in the TOC entry

---

I went with just `Integration` to mimic what Page Builder did in PWA.

---

👍